src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,45 @@
-export const Tabs = () => {};
+import className from "classnames";
+import { Tab } from "../../Tab";
+
+type Props = {
+  tabs: Tab[];
+  active: Tab;
```

Comment:
You will use only `content `property from active tab, so I suggest just to pass this content as `children` prop